### PR TITLE
Add hub translations

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?
Add the 4 hub keys to translations. To align eq-translations and runner I have had to add Gaelic. I removed Ulster Scots as it seems completely redundant, as there was a po and nothing else, however I updated the tests it had to Gaelic

### How to review 
Check the keys have been added correctly and laugh at my Welsh translations

### CODE REVIEW

File: doc/decisions/0001-context-in-translations.md [hunk 1]
```diff
@@ -18,6 +18,10 @@ For some translatable strings, we create a message context (`msgctxt`) that cont
   | `$.title` | Questionnaire title | |
   | `$.legal_basis` | Questionnaire legal basis | |
   | `$.messages` | Global answer error message | `message` rather than `messages` as each message is an individual translation |
+  | `$.hub.submission.button` | Custom hub submission button | |
+  | `$.hub.submission.guidance` | Custom hub submission guidance | |
+  | `$.hub.submission.title` | Custom hub submission title | |
+  | `$.hub.submission.warning` | Custom hub submission warning | |
```
Comment (ajmaddaford) [inline doc/decisions/0001-context-in-translations.md:24]: This file doesn't need to be updated as the most up to date list is in the Python.

File: tests/schemas/cy/test_language.json [hunk 2]
```diff
@@ -17,6 +17,15 @@
             "type": "string"
         }
     ],
+    "hub": {
+        "enabled": true,
+        "submission": {
+            "button": "Testun cyflwyno",
+            "guidance": "Canllawiau cyflwyno",
+            "title": "Arddangosir y teitl ar y canolbwynt pan fydd wedi'i gwblhau",
```
Comment (ajmaddaford) [inline tests/schemas/cy/test_language.json:25]:
```suggestion
            "title": "Arddangosir y teitl ar y canolbwynt pan fydd wedi’i gwblhau",
```

Comment (MebinAbraham) [reply tests/schemas/cy/test_language.json:25]: @ajmaddaford I can't quite remember what was agreed, but did we say that we would keep the `test_langauge` schemas in here and runner in sync?. If so, we should update the runner one, but obv that would break tests as is.

Comment (ajmaddaford) [reply tests/schemas/cy/test_language.json:25]: Yes, they should be kept in sync.

**Comment by LJBabbage** [comment]
I fixed and updated all the issue with test_languages (missing translation keys, empty values, old versions etc)for all languages only to find validator wouldn't allow me to have a final summary and a hub, so I had to add a separate schemas, doh. It seems daft not to include the changes to test_languages even thought they weren't in the end needed

File: eq_translations/schema_translation.py [hunk 3]
```diff
@@ -13,8 +13,8 @@ def load(self, translation_file_path):
         with open(translation_file_path, encoding="utf8") as translation_file:
             self.catalog = pofile.read_po(translation_file)
 
-            if self.catalog.locale_identifier == "sco_ulster":
-                self.catalog.locale = Locale("eo")
+            if self.catalog.locale_identifier == "ga_IE":
+                self.catalog.locale = Locale("ga")
```
Comment (MebinAbraham) [inline eq_translations/schema_translation.py:17]: Why was this changed?

Comment (LJBabbage) [reply eq_translations/schema_translation.py:17]: There was a random test_language-eo po, but no JSON to go with it, seemed completely pointless and as I had introduced a proper ga po I switched it over. It will also be easier for test schema translations if needed as Ulster Scots doesn't have one online

Comment (MebinAbraham) [reply eq_translations/schema_translation.py:17]: This was added to support Ulster scots in runner. Babel doesn't support Ulster scots but does support ga_IE hence `eo` is manually set.

Comment (LJBabbage) [reply eq_translations/schema_translation.py]: That does make sense, I weirdly thought this was in a test. I have also added a comment to clarify why we do it

File: tests/schemas/en/test_language.json [hunk 4]
```diff
@@ -17,13 +17,19 @@
             "type": "string"
         }
     ],
+    "submission": {
+        "button": "Submission text",
+        "guidance": "Submission guidance",
+        "title": "Submission title",
+        "warning": "Submission warning"
+    },
```
Comment (ajmaddaford) [inline tests/schemas/en/test_language.json:25]: This schema is slightly different to the one in runner - it's missing the hub enabled property and there are some minor formatting differences.

File: tests/schemas/cy/test_language.json [hunk 5]
```diff
@@ -17,8 +17,11 @@
             "type": "string"
         }
     ],
+    "hub": {
+        "enabled": true
+    },
     "submission": {
-        "button": "Testun cyflwyno",
+        "button": "Cnaipe aighneachta",
```
Comment (ajmaddaford) [inline tests/schemas/cy/test_language.json:24]: Not sure what happened here, but this looks like Gaelic rather than Welsh 😕 .

Comment (LJBabbage) [reply tests/schemas/cy/test_language.json:24]: They are both Welsh - Testun cyflwyno = Presentation text, Cnaipe aighneachta = Submission button.
